web_src/js/components/PullRequestMergeForm.vue

```diff
@@ -167,6 +169,7 @@ export default {
       this.deleteBranchAfterMerge = this.mergeForm.defaultDeleteBranchAfterMerge;
       this.mergeTitleFieldValue = this.mergeStyleDetail.mergeTitleFieldText;
       this.mergeMessageFieldValue = this.mergeStyleDetail.mergeMessageFieldText;
+      this.forceMerge = this.mergeForm.canMergeNow && !this.mergeForm.allOverridableChecksOk;
```

Comment:
IIRC canMergeNow and allOverridableChecksOk are all constants values in this component.

So the `forceMerge` assigning could be put in `mounted`, or just make it as a `computed` value.